docs/troubleshooting.md

```diff
@@ -1,6 +1,6 @@
 # Troubleshooting Marketplace Operator
 
-This document lists out some common failures related to the Marketplace Operator a user might encounter, along with ways to troubleshoot the failures. If you encouter a failure related to the Marketplace Operator that is not listed in this document, but should be, please open an issue or a PR to have the failure appended to this document. 
```

Comment:
I have a feeling this doc needs to be ripped out mostly, and point to the olm catalogsource troubleshooting doc instead. 

What'll be helpful instead, is a doc that talks about the OperatorHub API